lib/fizzy/instantiate.cpp

```diff
@@ -174,6 +174,13 @@ std::tuple<bytes_ptr, Limits> allocate_memory(const std::vector<Memory>& module_
     static const auto bytes_delete = [](bytes* b) noexcept { delete b; };
     static const auto null_delete = [](bytes*) noexcept {};
 
+    if (memory_pages_limit > MemoryPagesMaxLimit)
+    {
+        throw instantiate_error{"hard memory limit cannot exceed " +
```

Comment:
@gumb0 Is this now properly covered by spectests or you are planning to submit some cases?

Reply:
It's not covered and can't be, as "hard memory limit" is not defined in any precise way in the spec, it's just an internal limit which can be arbitrary set by implementations.

Reply:
It is not easy to find, but you can get this information from codecov. You need to go to single commit view -> Files, and then only select "spectests" flag.

https://codecov.io/gh/wasmx/fizzy/src/7d7bbfce98e8fca2927e539807beb8ffb4e8bac4/lib/fizzy/instantiate.cpp

![image](https://user-images.githubusercontent.com/573380/110621483-8f0ef180-819a-11eb-83a8-782ea235d313.png)

Reply:
> It is not easy to find, but you can get this information from codecov. 

I know, I went through that and listed the 10% missing cases in the chat last week. But that is only reflected if we upgrade out spectest branch, and we haven't done that in this PR :)